src/main/java/com/redislabs/RGHibernate.java

```diff
@@ -101,6 +100,9 @@ public void closeSession() {
   private void generateSession() {
     registry = new StandardServiceRegistryBuilder()
         .configure(InMemoryURLFactory.getInstance().build("configuration", xmlConf)).build();
+    //String dbpass = "1234";
+    //registry = new StandardServiceRegistryBuilder()
+     //       .configure(InMemoryURLFactory.getInstance().build("configuration", xmlConf.replaceAll("(connection\\.password\">)[^<]*", "$1" + dbpass))).build();
```

Comment:
Remove?

Reply:
Place holder for getting a password from external source.

Reply:
Removed this for now until we figure out what is the based way to handle password from external source